test_util/launch_configs/aws-linux-cluster.yaml

```diff
@@ -0,0 +1,13 @@
+---
+deployment_name: dcos-stack-that-was-provisioned-from-vpc
```

Comment:
This was an accidentally copy-pasted name; it should be `linux-cluster-made-by-dcos-launch` or something. This is *not* a configuration for dcos